processes/loc.py

```diff
@@ -7,10 +7,11 @@
 from mappings.loc import LOCMapping
 from managers import WebpubManifest
 from logger import createLog
+from datetime import datetime, timedelta
 
 logger = createLog(__name__)
 
-LOC_ROOT_OPEN_ACCESS = 'https://www.loc.gov/collections/open-access-books/?fo=json&fa=access-restricted%3Afalse&c=2&at=results'
+LOC_ROOT_OPEN_ACCESS = 'https://www.loc.gov/collections/open-access-books/?fo=json&fa=access-restricted%3Afalse&c=2&at=results&sp=2'
```

Comment:
curious about this change. It seems like below, you dynamically append `sp` to the url, so will this cause the url to have two `sp` values?

Reply:
Thanks for catching that. I was testing out the urls locally and forgot to get rid of the sp value I set there.